lib/Dialect/FIRRTL/Transforms/GrandCentral.cpp

```diff
@@ -1256,6 +1256,7 @@ bool GrandCentralPass::traverseField(Attribute field, IntegerAttr id,
             uloc,
             StringAttr::get(&getContext(), "assign " + path.getString() + ";"),
             ValueRange{}, ArrayAttr::get(&getContext(), path.getSymbols()));
+        numXMRs++;
```

Comment:
LLVM coding guidelines:  prefer prefix increment.

Reply:
Fixed in the `!fixup` commit.